situp-plugins/otel-trace-raw-processor/src/test/java/com.amazon.situp.plugins.processor.oteltrace/OTelTraceRawProcessorTest.java

```diff
@@ -44,6 +45,6 @@ public void testExportRequest() throws InvalidProtocolBufferException, JsonProce
         JsonFormat.parser().merge(sampleRequest, builder);
         final ExportTraceServiceRequest exportTraceServiceRequest = builder.build();
         final List<Record<String>> processedRecords = (List<Record<String>>) oTelTraceRawProcessor.execute(Collections.singletonList(new Record<>(exportTraceServiceRequest)));
-        assertThat(processedRecords.size()).isEqualTo(6);
+        Assertions.assertThat(processedRecords.size()).isEqualTo(6);
```

Comment:
nit: thought we were considering to use hamcrest over assertj; I am not advocating one over the other but just a question

Reply:
Ah. Looks like our plugins are still mostly using assertj. So maybe we leave it for separate PR?

Reply:
makes sense